software/audiobookshelf.yml

```
@@ -1,6 +1,6 @@
 name: Audiobookshelf
 website_url: https://www.audiobookshelf.org/
-description: Fully open-source self-hosted audiobook and podcast server. It streams all audio formats, keeps and syncs progress across devices. Comes with open-source apps for Android and iOS.
+description: Audiobook and podcast server. It streams all audio formats, keeps and syncs progress across devices. Comes with open-source apps for Android and iOS.
```

```suggestion
description: Audiobook and podcast server that streams all audio formats and keeps/syncs progress across devices. Comes with open-source apps for Android and iOS.
```
Optionally also remove "open-source", as I don't think it would be here unless it was.

---

well oops. Feel free to correct these in a separate PR since this one's been merged.